packages/p2p/src/components/my-profile/payment-methods/payment-methods-list/edit-payment-method-form.jsx

```diff
@@ -85,86 +87,101 @@ const EditPaymentMethodForm = () => {
-                                                <Text color='prominent' size='xs'>
-                                                    <Localize i18n_default_text='Choose your payment method' />
-                                                </Text>
-                                            }
-                                            value={my_profile_store.payment_method_to_edit.display_name}
-                                            required
-                                        />
-                                    )}
-                                </Field>
-                                {Object.values(my_profile_store.selected_payment_method_fields).map(
-                                    (payment_method_field, key) => {
-                                        return (
-                                            <Field
-                                                name={payment_method_field[0]}
-                                                id={payment_method_field[0]}
-                                                key={key}
-                                            >
-                                                {({ field }) => (
-                                                    <Input
-                                                        {...field}
-                                                        data-lpignore='true'
-                                                        error={errors[payment_method_field[0]]}
-                                                        type={
-                                                            payment_method_field[0] === 'instructions'
-                                                                ? 'textarea'
-                                                                : payment_method_field[1].type
-                                                        }
-                                                        label={payment_method_field[1].display_name}
-                                                        className='add-payment-method-form__payment-method-field'
-                                                        onChange={handleChange}
-                                                        name={payment_method_field[0]}
-                                                        required={!!payment_method_field[1].required}
-                                                    />
-                                                )}
-                                            </Field>
-                                        );
-                                    }
-                                )}
-                            </div>
-                            <div className='add-payment-method-form__buttons'>
-                                <Button
-                                    secondary
-                                    large
+                        <React.Fragment>
```

Comment:
Entire form is wrapped with Fragment to include a `PageReturn` component within the scope of Formik's state